src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciderConfig.yml

```diff
@@ -0,0 +1,12 @@
+# Order of action priorities
```

Comment:
Where will this yml be present on a deployed cluster? It cant be within the jar, users need to be able to replace it without building jars. Is that possible with current implementation? I think we should also have a path configuration which can be passed as jvm args in case users need to change its default location.

Reply:
I have modified the location to be in the same repo as the other `PA_CONFIGs`. I am thinking that this file can be thought of as a one stop config for all the decider related configs. Do we really want to give the knob of keeping all of them modifiable by the customer? Also, what benefit does changing the location brings? The action priority should be a one time setup from the customer during the cluster setup based on the type of the load customer anticipates and tuning after the setup. Once the initial setup is done Do we still anticipate these priorities changing?